tests/Unit/Resources/mails/html/clean-account.html

```diff
@@ -0,0 +1,109 @@
+    <table class="table table-mail" style="width: 100%; margin-top: 10px; -moz-box-shadow: 0 0 5px #afafaf; -webkit-box-shadow: 0 0 5px #afafaf; -o-box-shadow: 0 0 5px #afafaf; box-shadow: 0 0 5px #afafaf; filter: progid:DXImageTransform.Microsoft.Shadow(color=#afafaf, Direction=134, Strength=5);" width="100%">
+      <tr>
+        <td id="header-left" class="space" style="width: 20px;" width="20"> </td>
+        <td align="center">
+          <table class="table" bgcolor="#ffffff" style="width: 100%;" width="100%">
+                                        						<tr>
+							<td align="center" class="logo" style="border-bottom: 4px solid #333333;">
+								<a title="{shop_name}" href="{shop_url}" test="{shop_url}" style="color: #337ff1;">
+									<img src="{shop_logo}" alt="{shop_name}">
+								</a>
+							</td>
+						</tr>
+            
+            <tr>
+  <td align="center" class="titleblock">
+    <font size="2" face="Open-sans, sans-serif" color="#555454">
+      <span class="title" style="font-weight: 500; font-size: 28px; text-transform: uppercase; line-height: 33px;">Hi {firstname} {lastname},</span><br>
+      <span class="subtitle" style="font-weight: 500; font-size: 16px; text-transform: uppercase; line-height: 25px;">Thank you for creating a customer account at {shop_name}.</span>
+    </font>
+  </td>
+</tr>
+<tr>
+  <td class="space_footer" style="padding: 0;"> </td>
+</tr>
+<tr>
+  <td class="box" style="background-color: #f8f8f8; border: 1px solid #D6D4D4;" bgcolor="#f8f8f8">
+    <table class="table" style="width: 100%;" width="100%">
+      <tr>
+        <td width="10"> </td>
+        <td>
+          <font size="2" face="Open-sans, sans-serif" color="#555454">
+            <p data-html-only="1" style="margin: 3px 0 7px 0; text-transform: uppercase; font-weight: 500; font-size: 18px; padding-bottom: 10px; border-bottom: 1px solid #D6D4D4;">
+              Your {shop_name} login details
+            </p>
+            <span style="color: #777777;">
+              Here are your login details:<br>
+              <span style="color: #333333;"><strong>E-mail address: <a href="mailto:{email}" style="color: #337ff1;">{email}</a></strong></span>
+            </span>
+          </font>
+        </td>
+        <td width="10"> </td>
+      </tr>
+    </table>
+  </td>
+</tr>
+<tr>
+  <td class="space_footer" style="padding: 0;"> </td>
+</tr>
+<tr>
+  <td class="box" style="background-color: #f8f8f8; border: 1px solid #D6D4D4;" bgcolor="#f8f8f8">
+    <table class="table" style="width: 100%;" width="100%">
+      <tr>
+        <td width="10"> </td>
+        <td>
+          <font size="2" face="Open-sans, sans-serif" color="#555454">
+            <p style="margin: 3px 0 7px 0; text-transform: uppercase; font-weight: 500; font-size: 18px; padding-bottom: 10px; border-bottom: 1px solid #D6D4D4;">Important Security Tips:</p>
```

Comment:
Same here; if it needs to be translated, the whole paragraph should be localized. Let me know!

Reply:
This is a test file, so not too important But it was removed anyway so everything is fine ^^